### DISCUSSION TITLE
How to use background_enhancer ?

### DISCUSSION THREAD

**Discussion by AIhasArrived** [OpenTalker/SadTalker#739] [Q&A]
I tried answering the value "background_enhancer realesrgan" as explained here https://github.com/OpenTalker/SadTalker/blob/main/docs/best_practice.md
But got this error : ModuleNotFoundError: No module named 'realesrgan'
?

**Comment by U-DI-Page** [comment]
run the code in your terminal
`pip install realesrgan`
